README.md

```diff
@@ -125,6 +125,7 @@ bbgo pnl --exchange binance --asset BTC --since "2019-01-01"
 
 ## Advanced Configuration
 
+### Notification bot
```

Comment:
maybe just `Notification`? because for bot we might have interaction (in the future).

Reply:
done